designs/administrator/olm/operand-form-yaml-switching/index.md

```diff
@@ -1,26 +1,38 @@
 ---
 parent: Administrator
-version: 4.4
+version: 4.5
 ---
 
-# Operand (CR) Creation Form/YAML Switching
+# Operand (Custom Resource) Creation and Editing Form/YAML Switching
```

Comment:
There are part of these forms that we would like not shown when selected from the developer catalog.  For example, the breadcrumb is confusing to the developer.

Also, i'm not sure if the info on the right column is needed or helpful.  Have you guys gotten any feedback on that ?

Reply:
I do think the note that some fields might not be represented in the form is worth while, as perhaps the admin/dev might miss out on some field for the operand that is otherwise not represented in the form. Perhaps it could be reworded though?

Reply:
I talked  briefly with Tony Wu about the sidebar on the create operand form page. It was included for consistency with the Install Operator page.